POSserver/server/schemas/tag.py

```diff
@@ -24,8 +24,8 @@ def resolve_tags(self, info, **kwargs):
 
 class CreateTag(graphene.Mutation):
     class Arguments:
-        name = graphene.String()
-        description = graphene.String()
+        name = graphene.String(required=True)
+        description = graphene.String(required=True)
 
     ok = graphene.Boolean()
```

Comment:
Why is the Description field "required" for Tags?

Reply:
I believe we had the same question and removed the requirement. :)